setup.py

```diff
@@ -0,0 +1,38 @@
+# !/usr/bin/env python
+
+from distutils.core import setup
+
+try:
+    from setuptools import setup, find_packages  # noqa
+except ImportError:
+    from distutils.core import setup
+
+
+setup(
+    name='harview',
+    version='0.2.0',
+    description=(
+        'A commandline tool which takes as input a .har (HTTP Archive) file '
+        'and dumps a human-readable summary of it to the console'
+    ),
+    long_description='',
+    license='GPLv3',
```

Comment:
I'm not entirely sure what the License should be. I put this here, but let me know what you want me to update it to

Reply:
Hi Michael. Awesome work! The license should be MIT though. It's in the fine print all the way at the bottom of the README. I'll fix it my self and include a LICENSE file.